lib/services/authentication_api.dart

```diff
@@ -10,6 +10,10 @@ class AuthenticationApi {
       Uri.parse(serverUrlAuth + "/login"),
       headers: {'authorization': basicAuth},
     );
+
+    if (response.statusCode == 401) {
+      return "";
```

Comment:
Kan misschien gewoon null terugsturen
```suggestion
    if (response.statusCode == 401)  return null;
```